src/libs/Network.js

```diff
@@ -95,15 +98,33 @@ Onyx.connect({
     callback: val => email = val ? val.email : null,
 });
 
+/**
+ *
+ * @param {Boolean} val
+ */
+function setIsReady(val) {
+    isReady = val;
+}
+
 /**
  * Checks to see if a request can be made.
  *
  * @param {Object} request
+ * @param {String} request.type
+ * @param {String} request.command
  * @param {Object} request.data
  * @param {Boolean} request.data.forceNetworkRequest
  * @return {Boolean}
  */
 function canMakeRequest(request) {
+    // We'll not make the request now if the network is not ready (it should have higher priority than
+    // forceNetworkRequest and isQueuePaused because we wait until credentials + session objects have
+    // been read from persistent storage).
```

Comment:
Yeah, if we look at the code now it's easy to understand the priority because the context is still fresh in our minds, but maybe 6 months later could not be obvious for someone else (or even ourselves 😀). Also, I think that comment could help someone else (who wants to modify the `canMakeRequest`) to get up to speed quickly with the context. We can modify or remove the comment if `canMakeRequest` is changed again later :)

Reply:
>maybe 6 months later could not be obvious for someone else

On the other hand, all the context is here in GitHub. But if we are worried about this behavior changing it might be a good thing to write a test for.

Reply:
>We can modify or remove the comment if canMakeRequest is changed again later :)

I think the case I'm worried about is that we have to remember to update the comment (and it's very possible we will).

Reply:
I removed the comment and updated the unit test